src/Components/News/NewsList.js

```diff
@@ -1,32 +1,20 @@
-import React from 'react';
-import '../CardListStyles.css';
-
+import React from 'react'
+import '../CardListStyles.css'
 const NewsList = () => {
-    const newsMock = [
-        {id: 2, name: 'Titulo de prueba', description: 'Descripcion de prueba'},
-        {id: 1, name: 'Titulo de prueba', description: 'Descripcion de prueba'},
-        {id: 3, name: 'Titulo de prueba', description: 'Descripcion de prueba'}
-    ];
-
-    return (
-        <div>
-            <h1>Listado de Novedades</h1>
-            <ul className="list-container">
-                {newsMock.length > 0 ? 
-                    newsMock.map((element) => {
-                        return(
-                            <li className="card-info" key={element.id}>
-                                <h3>{element.name}</h3>
-                                <p>{element.description}</p>
-                            </li>
-                        )
-                    })
-                :
-                    <p>No hay novedades</p>
-                }
-            </ul>
-        </div>
-    );
+  const cardsFutureComponent = [{ id: 2 }, { id: 1 }, { id: 3 }]
+  return (
+    <div>
+      <h1>Novedades</h1>
+      <ul className='list-container'>
+        {cardsFutureComponent.length > 0 ? (
+          cardsFutureComponent.map((card) => {
```

Comment:
maybe you should use a material ui card and create a card component that then can be reused.

Reply:
this should be a function outside the view and then call the function here:
```
cosnt showCards = (lastNews) => 
    lastNews.map( news =>(<NewsCard key={news.id} name={news.name} description={news.description} />))
```